packages/gatsby/src/services/initialize.ts

```diff
@@ -586,6 +593,7 @@ export async function initialize({
     .map(
       plugin =>
         `{
+      name: '${plugin.name}',
```

Comment:
Added only to decorate thrown error with plugin name when there is exception during gatsby-ssr execution (like trying to use removed `loadPageDataSync` helper